devito/arch/archinfo.py

```diff
@@ -482,7 +488,7 @@ def _mro(cls):
 
     def _detect_isa(self):
         for i in reversed(self.known_isas):
-            if any(j.startswith(i) for j in get_cpu_info()['flags']):
+            if any(j.startswith(i) for j in as_tuple(get_cpu_info()['flags'])):
```

Comment:
`get_cpu_info().get('flag', None)`

Reply:
that is our `get_cpu_info`, which is guaranteed by construction to have a 'flags' entry